src/components/pages/pricing/hero/hero.jsx

```diff
@@ -20,14 +20,15 @@ const items = [
   {
     type: 'Free Tier',
     price: '$0 <span>/month</span>',
-    description: 'Generous always-available free tier, no credit card required.',
+    description: 'Run a free DB 24/7, no credit card required.',
```

Comment:
Let's try this since we're making the 24/7 clearer in the point below:

```suggestion
    description: 'Always-available free tier, no credit card required.',
```